src/sonic-config-engine/tests/sample_output/platform_output.json

```diff
@@ -810,7 +810,7 @@
     "index": "7",
     "lanes": "26,27",
     "description": "Eth7/2",
-    "admin_status": "up",
+    "admin_status": "down",
```

Comment:
Why is the test modified to have all port set to "down"?. I would expect test some port down and some up.

Reply:
Ports described in minigraph.xml as EthernetInterface has up admin_status:
Ethernet8
Ethernet9
Ethernet36
Ethernet98
Ethernet0
Ethernet6
Ethernet4
Ethernet109
Ethernet108